.github/workflows/static.yml

```diff
@@ -42,15 +48,18 @@ jobs:
         run: |
           cd $GITHUB_WORKSPACE/backend
           make gosec
-      - name: "Ensure 'cluster-agent' passes 'go-sec' - run 'make gosec' to identify issues"
+        - name: "Ensure 'backend' passes 'go-lint' - run 'make lint' to identify issues"
+        run: |
+          curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.40.1
+          cd $GITHUB_WORKSPACE/backend
+          make lint - name: "Ensure 'cluster-agent' passes 'go-sec' - run 'make gosec' to identify issues"
```

Comment:
Looks like the `-name:` field from the following line got mixed up into the `make lint` line.